fuel-vm/src/interpreter/executors/main.rs

```diff
@@ -239,11 +237,7 @@ where
                 self.debugger_set_last_state(program);
             }
 
-            let receipts_root = if self.receipts().is_empty() {
-                EMPTY_RECEIPTS_MERKLE_ROOT.into()
-            } else {
-                crypto::ephemeral_merkle_root(self.receipts().iter().map(|r| r.clone().to_bytes()))
-            };
+            let receipts_root = self.receipts.root();
 
             // TODO optimize
             if let Some(script) = self.tx.as_script_mut() {
```

Comment:
Could you remove "TODO" below, please? It seems https://github.com/FuelLabs/fuel-vm/issues/97 is solved=)